intergov/processors/channel_message_retriever/__init__.py

```diff
@@ -13,7 +13,7 @@
 
 class ChannelMessageRetrieveProcessor:
     def __init__(self):
-        channel_notification_repo_conf = env_queue_config('PROC_CHANNEL_NOTIFICATION_REPO')
+        channel_notification_repo_conf = env_queue_config('IGL_CHANNEL_NOTIFICATION_REPO')
```

Comment:
has @roleyfoley been informed? if not and we are not sure the change will be made immediately - could we determine the variable to use - first the new one, if it's empty then the old one?

Reply:
yes

Reply:
Updated in the hamlet deployment. Will go through when you merge this PR